kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -938,16 +938,11 @@ define([
      * @returns {object} with keys job status and values number of jobs in that state
      */
     function getCurrentJobCounts(jobsIndex, options = {}) {
-        if (
-            !jobsIndex ||
-            !Object.keys(jobsIndex).length ||
-            !jobsIndex.byId ||
-            !Object.keys(jobsIndex.byId).length
-        ) {
```

Comment:
make life easier by using `exec.jobs.byId` directly, rather than using `exec.jobs` as the argument and having an extra layer of annoying checks to ensure that the `byId` object is there, etc.